airflow/www/templates/airflow/tree.html

```diff
@@ -108,6 +109,22 @@
 var diagonal = d3.svg.diagonal()
     .projection(function(d) { return [d.y, d.x]; });
 
+const taskTip = d3.tip()
```

Comment:
graph.html and tree.html both extend from dag.html -- can this get defined instead as it is very similar between both of them.

Reply:
but all other pages also extend `dag.html`.